.github/workflows/vast.yaml

```diff
@@ -271,10 +277,7 @@ jobs:
           apt-get -y install ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb && \
           apt-get update
           apt-get -y install libarrow-dev libparquet-dev
-
-          # install CMake from pip -- we need at least 3.17 in CI for CCache
           python3 -m pip install --upgrade pip
```

Comment:
Is this still required now?

Reply:
I'm not sure if it is strictly required, but we use pip to install dependencies for the integration tests.

Reply:
That is always already upgrading pip (and also running inside a venv), so I think this can go.